# Changes needed to make OpenSP build on Solaris with the Studio 12.4 compilers.
# Since this is for our build, probably not something we can get upstream.

--- OpenSP-1.5.1/lib/MessageReporter.cxx.orig	2015-07-15 13:40:52.804057918 -0700
+++ OpenSP-1.5.1/lib/MessageReporter.cxx	2015-07-15 13:41:26.868348246 -0700
@@ -12,6 +12,7 @@
 #include "StorageManager.h"
 #include "macros.h"
 
+#include <stdlib.h>
 #include <string.h>
 
 
--- OpenSP-1.5.1/include/RangeMap.cxx.orig	2015-07-15 16:46:42.935477707 -0700
+++ OpenSP-1.5.1/include/RangeMap.cxx	2015-07-15 16:46:57.726685111 -0700
@@ -7,6 +7,7 @@
 #include "RangeMap.h"
 #include "ISet.h"
 #include "types.h"
+#include "constant.h"
 
 #ifdef SP_NAMESPACE
 namespace SP_NAMESPACE {
